README.md

```diff
@@ -1,93 +1,16 @@
-| __shared__ | Contains the source files for shared Observability content.|
-| __templates__ | Contains content templates.|
-
-## Contributing
-
-If you find any bugs in our documentation, or want to request an enhancement, then you can open an issue using our template. We also welcome contributions in the form of PRs. Before you submit a PR, make sure that you have signed our [Contributor License Agreement](https://www.elastic.co/contributor-agreement/).
-
-Contributing directly to the docs works differently across the doc sets in this repo.
-
-### Observability Guide
-
-The source files for the Observability Guide are written in [AsciiDoc](https://docs.asciidoctor.org/asciidoc/latest/) and are built using [elastic/docs](https://github.com/elastic/docs).
-
-To build the docs locally:
-
-1. Check out the `elastic/docs` repository, along with any repositories that contain source files.
-2. Run the `build_docs` script, passing in the path to the `index.asciidoc` and resource paths to other repos that contain source files. For example, to build the Observability Guide and open it in the browser, run:
-    ```
-    ../docs/build_docs --doc ./docs/en/observability/index.asciidoc --chunk 3 --resource ../apm-server --resource ../ingest-docs/docs --open
-    ```
-
-The above command assumes that this repo, [elastic/docs](https://github.com/elastic/docs), [elastic/ingest-docs](https://github.com/elastic/ingest-docs), and [elastic/apm-server](https://github.com/elastic/apm-server) are checked out into the same parent directory.
-
-If you prefer to use aliases, you can load the [elastic/docs/doc_build_aliases.sh file](https://github.com/elastic/docs/blob/master/doc_build_aliases.sh), which has the resources defined for you.
-
-### Elastic Observability Serverless docs
-
-The source files for the Serverless docs are written in [AsciiDoc](https://docs.asciidoctor.org/asciidoc/latest/) and are built using [elastic/docs](https://github.com/elastic/docs).
-
-To build the docs locally:
-
-1. Check out the `elastic/docs` repository, along with any repositories that contain source files.
-2. Run the `build_docs` script, passing in the path to the `index.asciidoc` and resource paths to other repos that contain source files. For example, to build the Observability Guide and open it in the browser, run:
-    ```
-    ../docs/build_docs --doc ../docs-content/serverless/index.asciidoc --chunk 5 --open --resource ./docs/en/serverless --resource ../security-docs/docs/serverless --resource ../docs-content/serverless
-    ```
+> [!IMPORTANT]
+> Starting with Elastic Stack v9.0.0 the content in this repo has been moved to other repos.
+>
+> **If you need to update the docs for version 8.17 or earlier**, you can open a PR in this repo targeting the specific version branch and backport to other branches using the [Backport CLI tool](https://github.com/sorenlouv/backport?tab=readme-ov-file#backport-cli-tool).
```

Comment:
I think this needs to either be (a) "**8.19** or earlier" or (b) "versions prior to 9.0".

Reply:
```suggestion
> **If you need to update the docs for a version prior to 9.0.0**, you can open a PR in this repo targeting the specific version branch and backport to other branches using the [Backport CLI tool](https://github.com/sorenlouv/backport?tab=readme-ov-file#backport-cli-tool).
```